torch/linalg/__init__.py

```diff
@@ -92,11 +91,22 @@
 """)
 
 inv = _add_docstr(_linalg.linalg_inv, r"""
-linalg.inv(input, *, out=None) -> Tensor
+linalg.inv(A, *, out=None) -> Tensor
+
+Computes the `inverse`_ of square matrix if it exists.
```

Comment:
"of square matrix" -> "of a square matrix"

"if it exists" -> "if it exists. Throws a runtime error if the matrix is not invertible." ?

Reply:
About the second point. I think it's better to leave it as it is. That line is going to be the short summary it appears when you click on `torch.linalg` (similar to the one that we have in `torch.nn`). As such, I think the description should be "short and sweet". Then the user can enter each documentation to see what happens in each particular case.